packages/node/src/dev-server.ts

```diff
@@ -212,7 +212,8 @@ async function compileUserCode(
         } catch (error) {
           // we can't easily show a meaningful stack trace
           // so, stick to just the error message for now
-          event.respondWith(new Response(error.message, {
+          const msg = error.cause ? (error.message + ': ' + (error.cause.message || error.cause)) : error.message;
```

Comment:
```suggestion
          const msg = error.cause
            ? (error.message + ': ' + (error.cause.message || error.cause))
            : error.message;
```

I think we should make this a bit clearer with line breaks or breaking apart the ternary.